luskan/src/intTest/java/org/opencadc/luskan/LuskanAsyncErrorTest.java

```diff
@@ -96,7 +96,7 @@
     public LuskanAsyncErrorTest()
     { 
         super(Constants.RESOURCE_ID);
-        File testCertFile = FileUtil.getFileFromResource("x509_CADCAnontest1.pem", LuskanSyncQueryTest.class);
+        File testCertFile = FileUtil.getFileFromResource("x509_CADCAuthtest1.pem", LuskanSyncQueryTest.class);
```

Comment:
intTest/README.md above mentions some generic cert names  but they are not used in the test setup (here and other places). That's just necessary to run other tests.

There should be an AuthQueryTest that verifies:
1. anon query fails as expected
2. query by luskan-test-noauth fails as expected
3. for completeness, the same query by luskan-test-auth succeeds

Reply:
Updated some int-tests to use the generic cert names and renamed the auth checking int-test to AuthQueryTest.